package.json

```diff
@@ -31,12 +31,13 @@
     "node": ">=0.10"
   },
   "dependencies": {
-    "lodash.uniq": "3.2.2"
+    "lodash.uniq": "3.2.2",
+    "lodash.repeat": "3.0.1"
   },
   "devDependencies": {
     "coveralls": "2.11.3",
     "dateformat": "1.0.11",
-    "eslint": "1.1.0",
+    "eslint": "1.9.0",
```

Comment:
@platinumazure i've updated the `peerDependency` of `eslint` to 1.4.0, since that is the first version which supports auto fixing.